assets/setup-wizard/index.js

```diff
@@ -26,19 +30,26 @@ registerSetupWizardStore();
  */
 const PARAM_NAME = 'step';
 
+/**
+ * A component to set the full screen and a custom class related to the current route.
+ */
+const Fullscreen = () => {
```

Comment:
It's a separate component just to have access to the QueryStringRouter provider.